srcpkgs/nvimpager/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'nvimpager'
+pkgname=nvimpager
+version=0.6
+revision=1
+wrksrc="${pkgname}-${version}"
+build_style=gnu-makefile
+make_build_args="PREFIX=/usr"
+make_build_target="nvimpager.configured"
+make_install_args="PREFIX=/usr"
+hostmakedepends="neovim curl git pandoc"
+depends="neovim>=0.3.0 bash"
+short_desc="Use nvim as a pager; view manpages, diffs, etc with syntax highlighting"
```

Comment:
I prefer this desc from the repo: "Use nvim as a pager to view manpages, diffs, etc", though I would change it to something like "Wrapper to use neovim as a pager to view manpages, diffs, etc".